src/core/webidl.js

```diff
@@ -38,9 +38,11 @@ const templates = {
       : // Other keywords like sequence, maplike, etc...
         hyperHTML`<a data-xref-type="dfn" data-cite="WebIDL">${keyword}</a>`;
   },
-  reference(wrapped, unescaped) {
+  reference(wrapped, unescaped, context = {}) {
```

Comment:
Oh, do we get undefined context sometimes? That's unexpected.

Reply:
No, that was my silliness from before I realized the WebIDL2.js hadn’t updated.